specifyweb/specify/datamodel.py

```diff
@@ -8202,7 +8204,149 @@
         fieldAliases=[
 
         ]
-    )
+    ),
+    Table( # CollectionObjectType
+        sp7_only=True,
+        django_app='specify',
+        classname='edu.ku.brc.specify.datamodel.CollectionObjectType',
+        table='collectionobjecttype',
+        tableId=1015,
+        idColumn='CollectionObjectTypeID',
+        idFieldName='collectionObjectTypeId',
+        idField=IdField(name='collectionObjectTypeId', column='collectionObjectTypeID', type='java.lang.Integer'),
+        fields=[
+            Field(name='name', column='Name', indexed=False, unique=False, required=True, type='java.lang.String', length=255),
+            Field(name='isdefault', column='IsDefault', indexed=False, unique=False, required=True, type='java.lang.Boolean'),
```

Comment:
The default CollectionObjectType is set in the `Collection` table

Reply:
Interesting, I thought were going to do a to-many side design (like isCurrent on determination). Setting on the collection can also work. Either way seems fine for now